opencart 1.5.6.4/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+
+services:
+  opencart:
+    build: ./
+    restart: always
+    ports:
+      - 8080:80
+    environment:
+      OPENCART_DB_HOST: opencart_db
+      OPENCART_DB_PASSWORD: examplepass
+    volumes:
+      - opencart:/var/www/html
+
+  opencart_db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_DATABASE: opencart
+      MYSQL_ROOT_PASSWORD: examplepass
```

Comment:
I took me some time to get all of the database credentials correctly inserted into the installation form. Is there something you can do to make it simpler? Is it possible to prefill the fields (especially useful for database server hostname), or change username, password and database name to `opencart`?

Reply:
The MySQL is overly difficult for me too. I will look into prefilling it

Reply:
I am currently using sed to replace MySQL credentials' placeholders in code. Let me know what you think of this solution.

Reply:
It is a bit hackish, but it'll have to do. If at some point we find a better solution then we can always change it.